cmp/grid/GridModel.js

```diff
@@ -479,6 +494,12 @@ export class GridModel {
         return c.children ? new ColumnGroup(c, this) : new Column(c, this);
     }
 
+    // Misc setters for visual/style observables.
+    @action setCompact(compact)                     {this.compact = compact}
+    @action setHighlightOnHover(highlightOnHover)   {this.highlightOnHover = highlightOnHover}
+    @action setRowBorders(rowBorders)               {this.rowBorders = rowBorders}
+    @action setStripeRows(stripeRows)               {this.stripeRows = stripeRows}
+
 
```

Comment:
why not just make these @bindable?

Reply:
I would be happy to, and had started out that way, but we had a pre-existing `@action setCompact()` and I recalled some discussion a while ago about whether or not we should use bindable for core toolkit classes such as GridModel that expose the setters as public API methods.  The fact that we didn't have bindable imported on the class led me to think we might be avoiding it on purpose.

I can see some advantage to having the explicit methods for IDE support as well, although I didn't add jsDoc comments (yet).  Having gone through all that, I think I slightly favor having these methods written out in the class, but don't feel very strongly.